src/app/components/search/AddFilterMenu.js

```diff
@@ -231,6 +232,18 @@ const AddFilterMenu = ({
         />
       ),
     });
+    options.push({
+      id: 'add-filter-menu__unmatched',
+      key: 'unmatched',
+      icon: <UnmatchedIcon />,
+      label: (
+        <FormattedMessage
+          id="addFilterMenu.unmatched"
+          defaultMessage="Media matched/unmatched"
```

Comment:
In order to make it consistent with the latest commit, I think that the label here should be just "Media unmatched" or "Media is unmatched", since we can't filter by "Media matched" using this filter anymore.

Reply:
oh right of course! updating